docker-compose.nexus.yml

```diff
@@ -0,0 +1,9 @@
+version: "3"
+services:
+  nexus:
+    image: "sonatype/nexus3:3.13.0"
+    ports:
+      - "8081:8081"
+    volumes:
+      - "./nexus-data:/nexus-data"
```

Comment:
In order to make this container accessible to `api-server` or `tile-server`, I added a named `network` that could be referenced in other files:

```diff
diff --git a/docker-compose.nexus.yml b/docker-compose.nexus.yml
index 0109fec7..9b66c20f 100644
--- a/docker-compose.nexus.yml
+++ b/docker-compose.nexus.yml
@@ -1,4 +1,4 @@
-version: "3"
+version: "3.5"
 services:
   nexus:
     image: "sonatype/nexus3:3.13.0"
@@ -7,3 +7,8 @@ services:
     volumes:
       - "./nexus-data:/nexus-data"
     restart: on-failure
+    networks:
+      - nexus
+networks:
+  nexus:
+    name: nexus
diff --git a/docker-compose.test.yml b/docker-compose.test.yml
index 49b9ff42..66e66123 100644
--- a/docker-compose.test.yml
+++ b/docker-compose.test.yml
@@ -75,6 +75,9 @@ services:
     build:
       context: ./app-backend
       dockerfile: Dockerfile.build
+    networks:
+      - "default"
+      - "nexus"
     env_file: .env
     volumes:
       - ./app-backend/:/opt/raster-foundry/app-backend/
@@ -86,3 +89,8 @@ services:
       - $HOME/.gnupg:/root/.gnupg
     working_dir: /opt/raster-foundry/app-backend/
     entrypoint: ./sbt
+networks:
+  default:
+  nexus:
+    external:
+      name: nexus
diff --git a/docker-compose.yml b/docker-compose.yml
index c8e35573..289cd266 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -80,6 +80,9 @@ services:
       postgres:
         condition: service_healthy
     env_file: .env
+    networks:
+      - "default"
+      - "nexus"
     environment:
       - RF_LOG_LEVEL=INFO
       - TILE_SERVER_LOCATION
@@ -134,6 +137,9 @@ services:
     external_links:
       - statsd
     env_file: .env
+    networks:
+      - "default"
+      - "nexus"
     environment:
       - RF_LOG_LEVEL=INFO
     ports:
@@ -183,3 +189,8 @@ services:
       - "-Dcom.sun.management.jmxremote.authenticate=false"
       - "-Dcom.sun.management.jmxremote.local.only=false"
       - "-Djava.rmi.server.hostname=localhost"
+networks:
+  default:
+  nexus:
+    external:
+      name: nexus
(END)
```